File: 1110 더하기 사이클/1110.cpp
#include <iostream>
int main() {
	int n;
	std::cin >> n;
	int origin = n, answer = 0;

	while (true) {
		n = ((n % 10) * 10) + (((n / 10) + (n % 10)) % 10);
		++answer;
		if (origin == n) {
			break;
		}
	}

	std::cout << answer << "\n";

	return 0;
}

File: 1110 더하기 사이클/1110.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString();

let n = parseInt(input);
let origin = n;
let answer = 0;
while (true) {
    n = ((n % 10) * 10) + ((parseInt(n / 10) + (n % 10)) % 10);
    ++answer;
    if (origin === n) {
        break;
    }
}

console.log(answer);